go.mod

```diff
@@ -15,7 +15,6 @@ replace (
 	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20190624125649-f0e46a78ea34
 	github.com/golang/protobuf => github.com/golang/protobuf v1.5.2
 	github.com/google/cadvisor => github.com/k3s-io/cadvisor v0.46.0-k3s1
-	github.com/googleapis/gax-go/v2 => github.com/googleapis/gax-go/v2 v2.0.5
```

Comment:
what does dropping this replacement have to do with bumping terratest? Something in these two changes seems to have made the linter angry.

Reply:
It looks like dropping this replacement took gax-go up to v2.7.0 which isn't right. Upstream is still on v2.1.1, ref:
https://github.com/kubernetes/kubernetes/blob/release-1.26/go.mod#L178

We should just update the pin to v2.1.1

Reply:
This is what I was referring to on slack, with regards to updating our pinned versions to track with upstream.

Reply:
When I attempted to bump terratest, I wanted a newer version of gax-go than we were giving it. I thought that it was the only thing pulling from it, but its not, its just the "first" in the list that go mod will tell me about. Reinstating the replacement works.

Reply:
It looks like we need to pin `google.golang.org/api v0.60.0` - ref: https://github.com/kubernetes/kubernetes/blob/release-1.26/go.mod#L88